src/core/caniuse.js

```diff
@@ -88,36 +87,29 @@ export async function run(conf) {
   await contentPromise;
 
   // remove from export
-  pub("amend-user-config", { caniuse: feature });
+  pub("amend-user-config", { caniuse: options.feature });
   sub("beforesave", outputDoc => {
     hyperHTML.bind(outputDoc.querySelector(".caniuse-stats"))`
       <a href="${featureURL}">caniuse.com</a>`;
   });
 }
 
 /**
- * Normalizes `conf.caniuse` object to hold normalized configuration
- *
+ * returns normalized `conf.caniuse` configuration
  * @param {Object} conf   configuration settings
  */
-function normalizeConf(conf) {
-  const DEFAULTS = {
-    maxAge: 60 * 60 * 24 * 1000, // 24 hours (in ms)
-    browsers: ["chrome", "firefox", "safari", "edge"],
-    versions: 4,
-  };
+function getNormalizedConf(conf) {
+  const DEFAULTS = { versions: 4 };
   if (typeof conf.caniuse === "string") {
-    conf.caniuse = { feature: conf.caniuse, ...DEFAULTS };
-    return;
+    return { feature: conf.caniuse, ...DEFAULTS };
   }
-  if (Array.isArray(conf.caniuse.browsers)) {
-    conf.caniuse.browsers = conf.caniuse.browsers
+  const caniuseConf = { ...DEFAULTS, ...conf.caniuse };
+  if (Array.isArray(caniuseConf.browsers)) {
+    caniuseConf.browsers = caniuseConf.browsers
       .map(b => b.toLowerCase())
       .filter(isValidBrowser);
```

Comment:
Could we only warn rather than filtering unknown browsers? Then users won't have to wait for ReSpec to add a new browser.

Reply:
Valid point. We iterate over the browsers in server response anyway, so this filter isn't needed.